site/public/apps/rheum-derm-immune-atlas/explorer/alternative-views.css

```
@@ -14,7 +14,9 @@
 .alt-link.selected{stroke:var(--text);stroke-width:3;opacity:1}.alt-link.context-muted,.bipartite-node-group.context-muted{opacity:.12}
 .difference-cell .difference-frame{fill:var(--panel);stroke:var(--line)}.difference-cell.shared .difference-frame{stroke:var(--good);stroke-width:3}.difference-cell.a-only .difference-frame{stroke:var(--accent);stroke-width:3}.difference-cell.b-only .difference-frame{stroke:var(--accent2);stroke-width:3;stroke-dasharray:5 3}.difference-cell.filtered .difference-frame{stroke:var(--muted);stroke-width:3;stroke-dasharray:2 4}.difference-cell.explicit-zero .difference-frame{stroke:var(--accent);stroke-width:3;stroke-dasharray:8 3}.difference-cell.unavailable .difference-frame{stroke:var(--faint);stroke-width:4;stroke-dasharray:1 3}.difference-cell.unresolved .difference-frame{stroke:var(--muted);stroke-dasharray:2 4}.difference-symbol{fill:var(--text);font:800 13px var(--font-mono)}
 .coverage-volume-grid{display:grid;grid-template-columns:minmax(0,1fr) 300px;gap:12px}.volume-overview svg{min-height:600px}.volume-axis{fill:none;stroke:var(--muted);stroke-width:1.5;marker-end:url(#volumeArrow)}.volume-plane{fill:var(--panel);stroke:var(--line);opacity:.72}.volume-point{fill:var(--good);stroke:var(--bg);stroke-width:1.5}.volume-point.derived{fill:var(--accent2)}.coverage-slice-shell{max-width:100%;overflow:auto;border:1px solid var(--line);margin-top:12px}.coverage-slice-grid{display:grid;min-width:max-content;background:var(--line);gap:1px;padding:1px}.coverage-slice-label,.coverage-slice-cell{min-width:28px;min-height:28px;background:var(--panel);display:grid;place-items:center}.coverage-slice-label{position:sticky;z-index:2;color:var(--muted);font:700 9px var(--font-mono)}.coverage-slice-label.row{left:0;min-width:150px;justify-content:start;padding:0 8px}.coverage-slice-label.column{top:0;writing-mode:vertical-rl;transform:rotate(180deg);min-height:150px;padding:6px 0}.coverage-slice-cell{border:0;cursor:pointer;color:var(--muted);font:800 9px var(--font-mono)}.coverage-slice-cell.direct{background:var(--good);color:var(--bg)}.coverage-slice-cell.derived{border:2px dashed var(--accent2);color:var(--accent2)}.coverage-slice-cell.explicit-zero{border:2px solid var(--accent);color:var(--accent)}.coverage-slice-cell.filtered{border:2px dotted var(--muted)}.coverage-slice-cell.unknown{color:var(--faint)}.coverage-slice-cell.structurally-unavailable{border:3px double var(--faint);color:var(--faint)}
+.mobile-visual-list,.mobile-visual-denominator{position:absolute;width:1px;height:1px;padding:0;margin:-1px;overflow:hidden;clip:rect(0,0,0,0);white-space:nowrap;border:0}
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)